### DISCUSSION TITLE
Help with encryption of the fields

### DISCUSSION THREAD

**Discussion by Waleedviews** [alpinejs/alpine#3506] [1. Help]
Hey everyone,

I am using alpine.js to load the data in the table, i want some of my data to be `encrypted` using `laravel facade`, is it possible to do something like below? 

` <a x-bind:href="'export/?exportid={{Crypt::encrypt('item.employee.user_id')}}'">Export</a>`

thank you.

**Comment by ekwoka** [comment]
Any encryption or whatever would happen on the backend.

Alpine wouldn't be concerned with that, as Alpine only runs in the browser.

It can neither encrypt or decrypt any data (securely) assuming you aren't using user provided/handled keys to do it BEFORE it gets to your server/after it comes from the server.

If you send Alpine encrypted data, it will simply treat it as a string. So it can do whatever strings do.

With this in mind, can you clarify more what your actual goal is for the real app (try not to reduce the case so much that it becomes useless)
